src/test/java/com/commercetools/sync/services/ProductServiceTest.java

```diff
@@ -0,0 +1,96 @@
+package com.commercetools.sync.services;
+
+import com.commercetools.sync.products.ProductTestUtils;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.products.Product;
+import io.sphere.sdk.products.ProductDraft;
+import io.sphere.sdk.products.commands.ProductUpdateCommand;
+import io.sphere.sdk.products.commands.updateactions.ChangeName;
+import io.sphere.sdk.products.commands.updateactions.Publish;
+import io.sphere.sdk.queries.PagedQueryResult;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Optional;
+
+import static java.util.Collections.singletonList;
+import static java.util.concurrent.CompletableFuture.completedFuture;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class ProductServiceTest {
+
+    private ProductService service;
+    private SphereClient ctpClient;
+
+    @Before
+    public void setUp() {
+        ctpClient = spy(SphereClient.class);
+        service = ProductService.of(ctpClient);
+    }
+
+    @Test
+    public void create() {
+        when(ctpClient.execute(any())).thenReturn(completedFuture(mock(Product.class)));
+
+        Product product = service.create(mock(ProductDraft.class)).toCompletableFuture().join();
+
+        assertThat(product).isNotNull();
```

Comment:
why do you think it doesn't make sense?
service's method has a certain signature which matters, provides implementation for some contract (even if not stated in javadoc yet), it has some logic, even if it's trivial. What are the cases when code should not be unit tested?